### PR TITLE
Update Link Mesheryctl Command Reference 

### LINKED ISSUE #2859: [Docs] Broken link: mesheryctl command reference
**Current State:**
The link the mesheryctl command reference at the bottom of this page is broken - https://docs.meshery.io/guides/mesheryctl/configuring-autocompletion-for-mesheryctl

**Desired State:**
Should point to `/reference/mesheryctl`

Screen Shot 2021-05-06 at 9 28 36 PM

---
**Contributor Resources**
- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)

### PR DESCRIPTION
**Pull request by Vi1234sh12**
Signed-off-by: Vishal Dhanure <vishald41234@gmail.com>

**Description**

This PR fixes #2859 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by Vi1234sh12** [comment]
@Nikhil-Ladha  Actually I'm Trouble solving this, can you help me

File: docs/pages/installation/kubernetes.md [hunk 1]
```diff
@@ -14,8 +14,8 @@ image: /assets/img/platforms/kubernetes.svg
 ## Available Deployment Methods
 
 - [Using `mesheryctl`](#using-mesheryctl)
-- [Using `helm`](#using-helm)
-- [Using Kubernetes manifests](#using-kubernetes-manifests)
+- [Using `helm-Charts`](#using-helm-Charts)
```
Comment (Nikhil-Ladha) [inline docs/pages/installation/kubernetes.md:17]: This change is not related to this issue.

**Comment by Nikhil-Ladha** [comment]
> @Nikhil-Ladha Actually I'm Trouble solving this, can you help me

What's the problem?

**Comment by Vi1234sh12** [comment]
> > @Nikhil-Ladha Actually I'm Trouble solving this, can you help me
> 
> What's the trouble?

Actually, I'm Currently only Update this issue   Mesheryctl Command Reference  but show me this error again

**Comment by Nikhil-Ladha** [comment]
> > > @Nikhil-Ladha Actually I'm Trouble solving this, can you help me
> > 
> > 
> > What's the trouble?
> 
> Actually, I'm Currently only Update this issue Mesheryctl Command Reference but show me this error again

What error?

**Comment by Nikhil-Ladha** [comment]
What command do you use to submit your commits?
You can follow the steps mentioned gere to fix the DCO issue https://github.com/layer5io/meshery/pull/2860/checks?check_run_id=2524637871

**Comment by Vi1234sh12** [comment]
> What command do you use to submit your commits?
> You can follow the steps mentioned gere to fix the DCO issue https://github.com/layer5io/meshery/pull/2860/checks?check_run_id=2524637871

I will try this  Command to submit  commits 
  ` commit -s -m “Update Link Mesheryctl Command Reference” `

**Comment by Vi1234sh12** [comment]
> > What command do you use to submit your commits?
> > You can follow the steps mentioned here to fix the DCO issue https://github.com/layer5io/meshery/pull/2860/checks?check_run_id=2524637871
> 
> I try this Command to submit commits
> `commit -s -m “Update Link Mesheryctl Command Reference”`

@Nikhil-Ladha  What should I do?

**Comment by Nikhil-Ladha** [comment]
Follow the steps mentioned in the link and update the PR.
Also, follow the [contribution guide](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow) and the [gitworkflow guide](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING-gitflow.md) to avoid these issues.
